main.go

```diff
@@ -18,13 +19,14 @@ import (
 func main() {
 	openFileFlag := flag.Bool("open", false, "To open the HTML report once generated")
 	fileNameFlag := flag.String("filename", "prettyplan-output.html", "Specify a filename other than prettyplan-output.html")
+	inputFlag := flag.Bool("input", false, "Whether to read the plan from stdin instead of running terraform")
```

Comment:
The name "input" is quite generic, I think `stdin` would at least be a more fit-for-purpose name for the flag here:

```suggestion
	inputFlag := flag.Bool("stdin", false, "Whether to read the plan from stdin instead of running terraform")
```

Reply:
Agreed, much better name.